Remove Application Registry and Connector Service from the Kyma documentation (#13984)
kyma-project/kyma: components/central-application-gateway/README.md

- 2 internal APIs implementing a proxy handler accessible via a service of type `ClusterIP`

### Standalone mode

The proxy API exposes the following endpoint:
```bash
{APPLICATION_NAME}/{SERVICE_NAME}/{TARGET_API_PATH}
``` 

For instance, if there's a `cc-occ-commerce-webservices` service in the `ec` Application CR, the user can send a request to the following URL: 
```bash
http://central-application-gateway.kyma-system:8080/ec/cc-occ-commerce-webservices/basesites
```

As a result, Central Application Gateway:
1. Looks for the `cc-occ-commerce-webservices` service in the `ec` Application CR and extracts the target URL path along with the authentication configuration.
2. Modifies the request to include the authentication data.
3. Sends the request to the following path:
   ```bash
   {TARGET_URL_EXTRACTED_FROM_APPLICATION_CR}/basesites
   ```

#### Standalone mode for Compass - simplified API

The standalone mode can also be used for Compass bundles with a single API definition.
In this case, `{API_DEFINITION_NAME}` is removed from the URL and the pattern looks as follows:
```bash
{APPLICATION_NAME}/{API_BUNDLE_NAME}/{TARGET_API_PATH}
```

> > **NOTE:** Invocation of service bundles configured with multiple API definitions will result in a `400 Bad Request` failure.

Reviewer: ```suggestion
> **NOTE:** Invocation of service bundles configured with multiple API definitions results in the `400 Bad Request` failure.
```

Author: Actually, I'd leave `a` here, because the underlying cause for this error can be different things, so in reality it's rather an error of the `400 Bad Request` sort.

Author: I'll edit your comment and apply the first part (`will result` > `results`).

## Usage

This section explains how to use Central Application Gateway.

### Start Central Application Gateway

To start Central Application Gateway, run this command:

> ```bash <!-- TODO: are we sure this is the command? It's the exact same as the one for the OLD App Gateway... -->

Author: It's OK

- **externalAPIPort** is the port that exposes the API which allows checking the component status. The default port is `8081`.
- **applicationSecretsNamespace** is the Namespace in which the Application secrets used by Central Application Gateway exist. The default Namespace is `kyma-integration`.
- **requestTimeout** is the timeout for requests sent through Central Application Gateway, expressed in seconds. The default value is `1`.
- **requestLogging** is the flag for logging incoming requests. The default value is `false`.
- **proxyTimeout** is the timeout for requests sent through the proxy, expressed in seconds. The default value is `10`.
- **proxyCacheTTL** is the time to live of the remote API information stored in the proxy cache, expressed in seconds. The default value is `120`.

## API

Central Application Gateway exposes:
- an external API implementing a health endpoint for liveness and readiness probes
- 2 internal APIs implementing a proxy handler accessible via a service of type `ClusterIP`

### Standalone mode

The proxy API exposes the following endpoint:
```bash
{APPLICATION_NAME}/{SERVICE_NAME}/{TARGET_API_PATH}
``` 

> <!-- TODO: verify 

Author: OK

http://central-application-gateway.kyma-system:8082/ec/cc-occ/commerce-webservices/basesites
```

As a result, Central Application Gateway:
1. Looks for the `cc-occ` service and the `commerce-webservices` entry in the `ec` Application CR and extracts the target URL path along with the authentication configuration.
2. Modifies the request to include the authentication data.
3. Sends the request to the following path: 
   ```bash
   {TARGET_URL_EXTRACTED_FROM_APPLICATION_CRD}/basesites
   ```

#### Handling ambiguous API definition names

> A combination of `{API_BUNDLE_NAME}` and `{API_DEFINITION_NAME}` which are extracted from an Application CR should be unique for a given application.

Reviewer: ```suggestion
A combination of `{API_BUNDLE_NAME}` and `{API_DEFINITION_NAME}` which are extracted from an Application CR must be unique for a given application.
```

> Invocation of endpoints with duplicate names will result in a `400 Bad Request` failure. In such a case, one of the names must be changed to avoid ambiguity.

Reviewer: ```suggestion
Invocation of endpoints with duplicate names results in the `400 Bad Request` failure. In such a case, change one of the names to avoid ambiguity.
```

Author: Again, I'll apply just the tense part. I'll also leave `must` because it puts emphasis on the required action.
